public/js/controllers/ui_controller.js

```diff
@@ -47,6 +47,63 @@ export class UIController {
+                <div class="d-flex justify-content-start m-2">
+                    <div class="dropdown">
+                        <button class="btn btn-brown shadow-none dropdown-toggle btn-sm mr-3" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Columns
+                        </button>
+                        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                            <form>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">Unique Texts</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">Unique Texts with a label</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">Done</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">Wrong Scheme messages</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">WS %</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">Not Coded messages</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input class="form-check-input" type="checkbox">
+                                    <label class="form-check-label">NC %</label>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+            
+                    <div class="ml-auto"><button class="btn btn-sm btn-brown shadow-none" id="reset">View All</button></div>
+                    <select class="btn-brown form-control form-control-sm shadow-none col-2 ml-2" id="keyword">
+                        <option disabled selected>Keyword Search</option>
+                        <option>COVID</option>
```

Comment:
populate this using active projects collection

Reply:
updated thanks.